keepercommander/commands/recordv3.py

```diff
@@ -1768,6 +1770,35 @@ def __init__(self, record_uid, record_title, paths):
         self.paths = paths
 
 
+def extract_otp_url(data):
```

Comment:
It looks like this code does not work for legacy records

Reply:
totp --details  
do not make any difference for me

Reply:
> It looks like this code does not work for legacy records

True - it currently makes no attempt to deal with legacy records.  Should it?

Reply:
> totp --details
> do not make any difference for me

You have to specify a record.

Reply:
I've added support for legacy records.